tests/unit/wmg_processing/test_marker_gene_cube_creation.py

```diff
@@ -26,9 +25,13 @@ def setUp(self):
 
     def test__marker_gene_cube_creation(self):
```

Comment:
add `@pytest.mark.skip(reason="no way of currently testing this")` to skip the test

Reply:
insert the github issue that will fix this in the reason